revel/util.go

```diff
@@ -92,6 +92,45 @@ func mustCopyDir(destDir, srcDir string, data map[string]interface{}) error {
 			return nil
 		}
 
+		// if this is a link, call mustCopyDir recursively on the actual directory
+		link, err := os.Lstat(srcPath)
+		if err == nil && link.Mode()&os.ModeSymlink == os.ModeSymlink {
+			// lookup the actual directory
+			realSrcPath, err := filepath.EvalSymlinks(srcPath)
+			panicOnError(err, "Failed to read symlink")
+
+			real_info, err := os.Stat(realSrcPath)
+			panicOnError(err, "Failed to stat symlink target")
+
+			// TODO should factor out this and the same code below for regular files
```

Comment:
Just had a look, It seems like contributor trying to say, repeated code lines needs to factor as method.